arches/app/models/graph.py

```diff
@@ -251,6 +251,11 @@ def check_default_configs(default_configs, configs):
 
                 self.populate_null_nodegroups()
 
+    @classmethod
+    def check(cls, **kwargs):
+        """Bypass GraphModel.check()."""
```

Comment:
Is this bypassing the `GraphModel.check()` method? I see it's being `super`-ed below

Reply:
Good Q. It is, because I passed `GraphModel` as the first arg to super instead of self. This calls GraphModel's super, not self's.